msteams-platform/concepts/build-and-test/teams-developer-portal.md

```diff
@@ -59,7 +59,7 @@ Follow the steps to import and manage your existing app in the Developer Portal.
 >
 > * The Developer Portal creates a unique app ID and locks the ID for your registered Teams app. You can’t edit or provide an ID of your choice, which prevents to have duplicate app IDs for multiple apps.
 > * If you create an app using the Microsoft Teams Toolkit for Visual Studio Code, you can manage your app in the Developer Portal. For more information, see [Build apps with teams toolkit and Visual studio code](~/toolkit/visual-studio-code-overview.md).
-> * You can import an existing app which you created on App Studio to the Developer Portal.
+> * You can import an existing app which you created on App Studio to the Developer Portal. To import an already published app to Developer Portal, the app owner need to raise a service request to transfer the ownership over the app ID. 
```

Comment:
Is app owner different from developer?
How to raise a service request?
What do you mean by transfer ownership over the app ID?

Reply:
Is app owner different from the developer? - App owner is the person who clicks on the New app button in the developer portal to create/build a new app. This person may or may not be the same as the Teams admin and other app owners. App ownership management is a unique concept to the Teams developer portal, where multiple users can be added to the same app to manage it effectively instead of having different local copies. At the time of adding the owner who is adding people can specify the role - administrative / operator, So a single app in TDP can have multiple owners and operatives who are just devs. 
How to raise a service request? - This is the same support request people raise from TAC/Message center. 
What do you mean by transfer ownership over the app ID? - In some cases(people more from app studio/ have built their app standalone, outside of TDP, there is no way to have a pre-defined Owner for the app. This is a lot deeper problem and you can read more about it here: https://microsoftapc-my.sharepoint.com/:w:/g/personal/srravic_microsoft_com/EdNOKQSqdYlGiZ8VyUEK3WgBW-sDRxtIjpkluzpu5DYang?e=Xq6dMG

If people still insist that they be added to an app as owners, TDP team will manually pick the request, try and validate the user raising the request and provide them with the means to be added as an owner to the app.